tools/Az.Tools.Predictor/Az.Tools.Predictor/Telemetry/AzPredictorTelemetryClient.cs

```diff
@@ -80,7 +78,7 @@ public virtual void OnHistory(HistoryTelemetryData telemetryData)
         {
             PostTelemetryData(telemetryData);
 
-            _commandId = Guid.NewGuid().ToString();
+            CommandId = Guid.NewGuid().ToString();
```

Comment:
if commandid already have a guid in property init place, then here we will force to refresh the guid again?

Reply:
Yes. Those Ids are updated once the cycle ends. CommandId is updated when a command is executed. RequestId is updated for a new request to the server etc.